src/WebKit_Performance_011.test.js

```diff
@@ -0,0 +1,83 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  webKitBrowserActions,
+  settingUrl,
+  isUrlLoaded,
+} from './commonMethods/commonFunctions'
+import constants from './commonMethods/constants'
+
+const URL = 'http://peacekeeper.futuremark.com/run.action'
+const resultURL = 'http://peacekeeper.futuremark.com/results'
+
+export default {
+  title: 'WPEWebkit Peacekeeper Benchmark test',
+  description: 'Loads the Peacekeeper Benchmark test and get the results',
+  steps: [
+    {
+      description: 'Deactivating monitor',
```

Comment:
Same here, artifact from older implementations of WPEFramework. I think we can remove the monitor activate/deactivates.

Reply:
Updated as per the comment